packages/atlas/src/api/hooks/comments.ts

```diff
@@ -29,6 +32,39 @@ export const useComments = (
   }
 }
 
+export const useComment = (
+  id: string,
+  memberId: string,
+  videoId: string,
+  opts?: QueryHookOptions<GetCommentQuery, GetCommentQueryVariables>
+) => {
+  const { data, ...rest } = useGetCommentQuery({
+    ...opts,
+    variables: { where: { id }, memberId, videoId },
+  })
+
+  const userCommentReactionsLookup =
```

Comment:
We do this twice, maybe we could create a function which takes comment reactions and convert them into lookup?